### PR TITLE
Use react-native-animateable-text instead of TextInput for numbers

### PR DESCRIPTION
**Pull request by savelichalex**

### CODE REVIEW

File: packages/hydrogen/src/Numbers/AnimateableText.web.tsx [hunk 1]
```diff
@@ -35,12 +37,13 @@ class TextInputImitator extends React.Component<TextInput['props']> {
...
-                {this.props.defaultValue}
+            <Text ref={this._ref as any} style={style}>
+                {text}
             </Text>
         );
     }
 }
 
-export const AnimatedTextInput = Animated.createAnimatedComponent(TextInputImitator);
+export const AnimateableText = Animated.createAnimatedComponent(TextInputImitator);
```
Comment (sertony) [inline packages/hydrogen/src/Numbers/AnimateableText.web.tsx]: 👍 Great compat

File: packages/localization/src/constants.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const languagesInfo: Record<Language, LanguageInfo> = {
         dayJS: 'fr',
     },
     [Language.PtBR]: {
-        name: 'Português (Br)',
+        name: 'Português brasileiro',
```
Comment (savelichalex) [inline packages/localization/src/constants.ts:25]: Sorry for that, forgot to put it in another PR

**Review by sertony — APPROVED** [review]

**Review by SAnatoliiS — APPROVED** [review]
